lesson_2_Section_30_new.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>I think you dont necessarily have to be 5 for the analogy to work right?  I think if you're an impressionist painter drawing the line you'd probably get the same result as a 5 year old.  There could be a small chance it comes across a bit patronizing.  Overall though I like your point.  Maybe like, "Wether you're 5 years old or an *insert fancy type of artist here*, you don't think about drawing like, here's a fixed function, now let's mentally minimize the residual"? though that's maybe too wordy.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
good point, ill change this to impressionist painter, itl make for a better thing add as a fun image anyway